src/App.jsx

```diff
@@ -5,18 +5,19 @@ import axios from 'axios';
 
 import RecipeBrief from './components/RecipesDisplay/RecipeBrief';
 import Header from './components/header/Header';
+import SearchBar from './components/searchBar/SearchBar';
 
 function App() {
   // I'm thinking setIngredients will be passed as props to the search bar
-  const [ingredients, setIngredients] = useState('');
   const [recipes, setRecipes] = useState(null);
-  const [isSearching, setIsSearching] = useState(false);
+  // const [isSearching, setIsSearching] = useState(false);
   const [error, setError] = useState('Add an ingredient to search!');
 
   // This function could potentially be moved out to a "utils" folder?
-  // In that case, I think the response or error could be returned, and then used with setRecipes or setError here
+  // In that case, I think the response or error could be returned,
+  // and then used with setRecipes or setError here
   async function fetchData(ingredientString) {
```

Comment:
This is an api fetching function and should be refactored from this layer. This file does not specifically focus on this layer of logic. It should be exported from an API file.